### ISSUE TITLE
As a devops, I want to deploy a separate PostgreSQL instance for Fertiscan so we can have full control over the database environment

### ISSUE THREAD

**Issue by k-allagbe** [ai-cfia/ailab-datastore#162]
## Description

**Context**  
Currently, Fertiscan's database shares the same PostgreSQL deployment as Nachet, which is in a deployment we do not control.

**Problem Statement**  
We need to have a dedicated PostgreSQL instance for Fertiscan deployed in our own resource group. This would give us full control over database performance, scalability, security and costs monitoring while eliminating external dependencies on Nachet's infrastructure.

## Acceptance Criteria

- A dedicated PostgreSQL instance is deployed in our own resource group.
- Fertiscan is fully migrated to the new PostgreSQL instance without data loss.
- All existing connections to the database are updated to point to the new instance.

**Comment by Francois-Werbrouck** [comment]
@ThomasCardin I created what I thought would be the TODO list of what is needed to pursue this issue. Please add/modify depending on the Acceptance Criteria

**Comment by Francois-Werbrouck** [comment]
@k-allagbe I'm curious, do you plan on making a follow up with FSDH to explain why we are making the switch? They might be inclined into helping us solve our issues instead of us managing our own ressources.

Also, could you develop on what you meant by `this would give us full control over database performance, scalability, security and costs monitoring`. FSDH is giving us cost monitoring services. Also having control over our database comes with a share of responsibility now our plate. I'm not inclined to say this would make our database more secure, I'd say its quite the opposite. 

I'm also curious on what you mean by `while eliminating external dependencies on Nachet's infrastructure.` since Fertiscan db has no dependencies on Nachet. They both are in a seperate Database.

Dont forget FSDH is a governemental branch made to support scientific projects and ease the creation of our solutions. When we began creating the datastore, I was attending their weekly meeting and also exchanged with them through emails to solve previous issues on their DB services to better suits their client's needs. 

I'm asking all these question to find the value of this issue and to justify the need to increase our load of responsability, while using ressources to solve this issue (since me and thomas are assigned to this).

**Comment by nmeraihi** [comment]
I agree with @k-allagbe  on this issue. It’s essential that we move towards a solution that ensures the independence of our project from other entities, which will provide us with greater flexibility in the long term. Especially to bring everything into our production environment.

**Comment by Francois-Werbrouck** [comment]
Sounds good to me! We will proceed with the requiered changes to deploy both project into our own server 🚀

**Comment by k-allagbe** [comment]
@Francois-Werbrouck one postgres instance should be enough. Also we have a resource group dedicated to fertiscan where we'll deploy the instance. This means that nachet should probably have it's own postgres instance or keep using fsdh's. Please sync with @RussellJimmies on the matter.

**Comment by Francois-Werbrouck** [comment]
I'm unsure about your definition of postgres instance, but what I'm suggesting is having a server for Test and another for Prod, to create 2 different environment. Which I believe is the reason we are pushing for those changes.
> Especially to bring everything into our production environment.

 With those two server we can then deploy each project individually in their distinct db. Not sure about what the problem is. 

Also if we are moving from fsdh services to our own, I suggest we do on both project in parallel do avoid duplicating the work later on. Keeping all our project on the same service also help us manage them in the long run instead of managing 2 different service.

**Comment by k-allagbe** [comment]
By postgres instance, I mean a postgres deployment (or resource). 

I suggest one instance for each project, with the fertiscan's deployed in rg-fertiscan. In the fertiscan's instance we can create 2 dbs for prod and dev. We created rg-fertiscan and I believe another one for Nachet, for monitoring resources and managing costs. If we want to achieve that effectively, we should not have multiple projects use the same resources.

> I suggest we do on both project in parallel

I'm not aware of Nachet's development but, by all means.

Moving this issue to the devops repo.
